src/calculating.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const calculating = (number, answer) => {
+  const bulls = [];
+  const cows = [];
+
+  const len = answer.toString().length;
+
+  for (let i = 0; i < len; i++) {
+    const pcValue = number.toString().split('')[i];
+    const inputValue = answer.toString().split('')[i];
+
+    if (pcValue === inputValue) {
+      bulls.push(inputValue);
+    } else if (pcValue !== inputValue) {
+      cows.push(inputValue);
```

Comment:
This logic only checks whether the digits are the same and in the same position, but does not properly calculate the `cows` according to the task (number of digits in the guessed number that are in the correct number but the wrong place).